modules/clients/pom.xml

```diff
@@ -126,6 +126,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
This dependency is required for multi-JVM test, isn't it?
If so, it is ok for me.

Reply:
Sure. `startGridRemote` uses xstream by `IgniteNodeRunner#storeToFile`